app/serializers/api/participant_serializer.rb

```diff
@@ -0,0 +1,50 @@
+module API
+  class ParticipantSerializer < Blueprinter::Base
+    class AttributesSerializer < Blueprinter::Base
+      exclude :id
```

Comment:
I was wondering if we should be querying by teacher, as that's the main id similar to user on ecf1?